ui/dashboards/src/components/PanelDrawer/PanelPreview.tsx

```diff
@@ -34,9 +37,30 @@ export function PanelPreview({ name, description, kind, spec }: PanelEditorValue
     return null;
   }
 
+  // ex: markdown panel does not have a query
+  if (!queries) {
+    return (
+      <Box height={300}>
```

Comment:
Can this height value get pulled out into a constant now that you're using it in multiple places?